lib/decrypt.rb

```diff
@@ -0,0 +1,15 @@
+require 'date'
+require './lib/enigma'
+
+handle = File.open(ARGV[0], "r")
+encrypted = handle.read
+handle.close
+
+enigma = Enigma.new
+decrypted = enigma.decrypt(encrypted, ARGV[2], ARGV[3])
+
+writer = File.open(ARGV[1], "w")
+writer.write(decrypted[:decryption])
+writer.close
+
```

Comment:
The same as encrypt.rb except it uses decrypt and needs a key and date from the command line.